### DISCUSSION TITLE
Crash logging

### DISCUSSION THREAD

**Discussion by jbcullis** [microsoft/react-native-windows#10918] [Q&A] [answered]
After updating to .70 I'm getting some feedback on the app freezing when expanding from being minimized.
I'd like to add some logging in at the native level, is there an easy entry point to catch all exceptions?

**Comment by chrisglein** [comment, marked as answer]
For this scenario you could try setting tracing in `OnActivated`.

Not sure if you can do this for the global exception catching, but something useful for this kind of scenario is [Tracepoints](https://learn.microsoft.com/en-us/visualstudio/debugger/using-tracepoints?view=vs-2022). But that's for live debugging, and it sounds like you can't repro this situation? And that's what you're trying to narrow down?

**Reply by jbcullis** [reply]
@chrisglein just trying to wire up logging to make sure we're not missing anything.
I'll give it a shot.
